### ISSUE TITLE
Cannot See any warnings

### ISSUE THREAD

**Issue by Karachristou** [celian-rib/vscode-webvalidation-extension#22]
It does not give me the warnings, only the errors. I would like to have an option to see the warnings too,

**Comment by celian-rib** [comment]
Hi @Karachristou thanks for submitting this issue, the extension should give you warnings just like that :

![image](https://user-images.githubusercontent.com/38594012/152410659-ae30a458-e939-4413-9a7b-07c535db43c3.png)

> Here is the html I have been using to test it :
```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <header>
        <!-- Warning below  -->
        <h2>

        </h2>
    </header>
    
</body>
<!-- Error below  -->
<script type="javascript">

</script>
</html>
```

Is the behaviour different on your side ?

**Comment by Karachristou** [comment]
@celian-rib  thanks for your response, the behaviour works like you depicted for HTML, however no warning appear for me in CSS code.

**Comment by celian-rib** [comment]
> @celian-rib thanks for your response, the behaviour works like you depicted for HTML, however no warning appear for me in CSS code.

Do you have any CSS as an example on which the the problem occurs?
